cmd/gaia/cli_test/test_helpers.go

```diff
@@ -426,11 +426,18 @@ func (f *Fixtures) QueryGovParamTallying() gov.TallyParams {
 }
 
 // QueryGovProposals is gaiacli query gov proposals
-func (f *Fixtures) QueryGovProposals(flags ...string) string {
+func (f *Fixtures) QueryGovProposals(flags ...string) gov.Proposals {
 	cmd := fmt.Sprintf("gaiacli query gov proposals %v", f.Flags())
 	stdout, stderr := tests.ExecuteT(f.T, addFlags(cmd, flags), "")
+	if stderr == "ERROR: No matching proposals found" {
```

Comment:
hmmm, this seems somewhat dangerous. May I suggest that we do a `strings.Contains()`?

Reply:
agreed, it was hacky. fixed.